hearth/lib/hearth/http/field.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Hearth
+  module HTTP
+    # Represents an HTTP field.
+    # @api private
+    class Field
+      # @param [String] name The name of the field.
+      # @param [Array|#to_s] value (nil) The values for the field. It can be any
+      #   object that responds to `#to_s` or an Array of objects that respond to
+      #  `#to_s`.
+      # @param [Symbol] kind The kind of field, either :header or :trailer.
+      #   Trailers are currently not supported by Net::HTTP.
```

Comment:
This seems like a detail of Net::Http, not of `Field`.... but I guess since these are api private, its fine.

Reply:
I figure it was a reasonable callout. I can remove it.